utils.py

```diff
@@ -385,3 +386,15 @@ def reverse_complement(sequence):
     """
     rev_matrix = str.maketrans("atgcATGC", "tacgTACG")
     return sequence.translate(rev_matrix)[::-1]
+
+
+def check_file(file_path: pathlib.Path):
```

Comment:
Let's also schedule some time to go through the whole code together to see what else should be added to utils.py... I like what you've done so far! :)